reviews/review-02+2020-11-11.md

```diff
@@ -0,0 +1,77 @@
+## What do you think is good about this manuscript?
+
+The range of topics covered is thorough without being overwhelming.
+There is a tendency to attempt to cram every single aspect of software
+engineering into a book like this without acknowledging feasible
+limits. Yes, it is over 450 pages in it's current draft, but that
+seems a reasonable amount given the intended audience.
+
+## What reservations do you have about this manuscript?
+
+There is a sense in which the intended audience may not view
+themselves as software engineers (i.e., they are likely researchers in
+some other specific scientific discipline). As such, the term
+"research software engineering" may miss the mark. That said, there
+are plenty of people who do use this term and there are not really any
+effective alternatives in common use. So the title is still likely to
+reach it's intended audience.
+
```

Comment:
I think the Introduction does a good job of describing what a "research software engineer" is, so a reader doesn't have to get very far to figure out if this is for them, but I still think this is an interesting point.  Are we missing out on people who aren't familiar with the term "research software engineering", or don't recognize what they do (or want to do is "research software engineering"?

Is it worth giving our title one last chunk of thought?  I'm not a huge fan of colons, but maybe a subtitle might help.  Stealing straight from the Intro: "Building software that makes research possible"

Reply:
I'm also a little worried about our use of the term "research software engineer," but I'm also not sure that there is a better alternative. 

From my limited participation in eResearch conferences and national RSE associations, there seems to be two paths by which people come into the research software space:
1. Researchers who pick up the software skills later on
2. Software engineers who pick up the research knowledge later on

The latter would definitely call themselves "Research Software Engineers" and in my experience they tend to dominate the active membership of national RSE associations. Our book isn't for these people. Our book is for researchers who are trying to pick up software skills, who are less likely to identify with the RSE label. Perhaps we need to make this distinction clearer in the book prologue?

An alternative book title might do away with the word engineering and just be: 
"Writing Research Software with Python"

Reply:
I'd like to keep RSE in the title - the term has some traction, and while people who already have these skills may dominate the associations, I think there's a much larger (quieter) group who are trying to acquire them and become RSEs. I share @cwickham's dislike of colons, but I like the subtitle idea a lot.